src/hotspot/cpu/x86/macroAssembler_x86.hpp

```diff
@@ -1815,6 +1815,8 @@ class MacroAssembler: public Assembler {
   void mov_metadata(Register dst, Metadata* obj);
   void mov_metadata(Address  dst, Metadata* obj, Register rscratch);
 
+  void mov_ptrslot(Register dst, intptr_t val);
```

Comment:
I believe the convention here would be `movptr_imm64`

Reply:
Good name, renamed.